decentraland/unity-renderer
# fix: Desynchronization between Settings Panel and Voice Chat window

## What does this PR change?
Fix #2896

The "Allow voice" filter was working different when it was set from the **Settings Panel (wrong behaviour)** than when it was set from the own **Voice Chat window (correct behaviour)**.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/voice-chat-allow-filter-issues&position=150,120 in 2 browsers: one with a wallet account and another one as a guest.
2. Change the "Allow voice chat" filter from these both places: Settings Panel and Voice Chat HUD header. Do it from each one of the users.
3. Check the reception of audio is working as expected depending on what filter is selected.

### Note
Please, follow also the steps to reproduce of the issue and check that it is already fixed: #2896

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
